src/AutoPlay.ts

```diff
@@ -98,7 +105,7 @@ class AutoPlay implements Plugin {
     // DO-NOTHING
   }
 
-  public play = () => {
+  public play = (e: MoveEndEvent | SelectEvent | null, duration?: number) => {
```

Comment:
Why does play need an event?

Reply:
The following implementation exists in the init process of the plugin.
https://github.com/naver/egjs-flicking-plugins/blob/master/src/AutoPlay.ts#L58-L63
```ts
    flicking.on({
      [EVENTS.MOVE_START]: this.stop,
      [EVENTS.HOLD_START]: this.stop,
      [EVENTS.MOVE_END]: this.play,
      [EVENTS.SELECT]: this.play
    });
```
As a result, MoveEndEvent or SelectEvent is passed as the first argument of `play`.
I thought of two ways to deal with this.
- Separate a new function that calls `prev` or `next`. This function takes a duration as an argument. And inside the function following the MOVE_END and SELECT events, the above new function is called.
- Into the existing function following the existing MOVE_END and SELECT events, add a new argument `duration` as the second argument.

I took the second way, thought that it would be weird to separate the `play` into smaller functions.

Reply:
I think this makes it looks weirder, there're two problems here.

First, it's changing the spec of Autoplay's "public" method `play`.
When someone was using `autoplay.play()`, without any arguments in their code, this will not break the behavior but will throw a type error.

Second, the first argument(event) is doing nothing at all.
`play` is not intended to be used only by attaching it as an event handler, we're just using like that internally.
Suppose someone is calling `Autoplay.play()`, that one has to use the unknown `null` as the first parameter always.
Refactoring the code is up to you, but I recommend you separate the methods for this case.

Reply:
Considering that `play` is a public method, it would be better to use the way I first thought of.
After adding `duration` optional to the public method `play`, previous implementation was modified.